src/main/scala/com/gu/newsletterlistcleanse/db/Campaigns.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.newsletterlistcleanse.db
+
+import scalikejdbc._
+import scalikejdbc.athena._
+
+trait Campaigns {
+  def fetchCampaignSentDates(campaignNames: List[String]): List[CampaignSentDate]
+}
+
+class CampaignsFromDB extends Campaigns {
+  override def fetchCampaignSentDates(campaignNames: List[String]): List[CampaignSentDate] = {
+    DB.athena { implicit session =>
+      sql"""
+        SELECT campaign_name, campaign_id, timestamp FROM (
+          SELECT row_number() over(partition by campaign_name) AS rn, *
+          FROM (
+            SELECT
+              campaign_name,
+              campaign_id,
+              timestamp
+            FROM
+              "clean"."braze_dispatch"
+            where campaign_name in ($campaignNames)
+            order by timestamp desc
+          )
+        )
+        WHERE rn <= 94
```

Comment:
Maybe 94 should be a const passed into this query. We could include it in the Newsletter object (even calculate it from the cleansing policy map), so that we avoid inconsistency between the 2 places in the future.